apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -2,26 +2,22 @@
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
   [class]="customClass">
   <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+    [src]="imageSrc"
+    width="200px"/>
 
   <section>
     <app-list-item
       *ngFor="let item of list"
-      [name]="item.firstname"
-      [id]="item.id"
-      [type]="type">
+      [name]="item.name!">
```

Comment:
not very customizable, what will happen if you need a button for teacher card only. Look at `ngTemplateOutlet`